src/clothing_mod.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#ifndef CLOTHING_MOD_H
+#define CLOTHING_MOD_H
+
+#include <stddef.h>
+#include <string>
+#include <vector>
+
+#include "type_id.h"
+#include "string_id.h"
+#include "ui.h"
+#include "output.h"
```

Comment:
This one and the above appear to be unused. If so, remove them.

Reply:
"type_id.h" is needed. others are unused.